src/main/java/ee/tuleva/onboarding/account/AccountStatementService.java

```diff
@@ -25,14 +25,17 @@
 
     return accountStatement.stream()
         .filter(fundBalanceDto -> fundBalanceDto.getIsin() != null)
-        .filter(
-            fundBalanceDto ->
-                compare(ZERO, fundBalanceDto.getValue()) != 0
-                    || fundBalanceDto.isActiveContributions())
+        .filter(this::secondPillarNotEmpty)
         .map(fundBalanceDto -> convertToFundBalance(fundBalanceDto, person))
         .collect(toList());
   }
 
+  private boolean secondPillarNotEmpty(FundBalanceDto fundBalanceDto) {
```

Comment:
`boolean` function names should start with `is...` - i.e. `isSecondPillarNotEmpty` in this case.